assets/js/view/game_card.rb

```diff
@@ -66,7 +66,7 @@ def render_header
           end
           JOIN_YELLOW
         when 'active'
-          buttons << render_button('Enter', -> { enter_game(@gdata) })
+          buttons << h('a.button', { style: { top: '1rem', float: 'right', 'text-decoration': 'none', }, attrs: { href: url(@gdata) } }, 'Enter')
```

Comment:
let's make this a function so we can do it properly

override the onclick handler to return false and provide a custom on click. that way it doesn't actually do a refresh

Reply:
I am not aware of an accessible cross-browser compatible way to have a button that behaves like the user would expect with ctrl/cmd/middle/etc-click to open a new tab, but also does a page load without refresh if they click/enter the button normally.